clients/README.md

```diff
@@ -106,10 +106,10 @@ After making the changes, the first step is to run the local node.
 cargo run --bin spacewalk-standalone --release -- --dev
 ```
 
-Once the local node is running, run this command:
+Once the local node is running, run this command from the clients directory:
```

Comment:
Nice, can we change the command in line 92 to specify installation of a fixed subxt version? Something like `cargo install subxt-cli@0.31.0`